### ISSUE TITLE
GooglePayTrnRegistrar not executing exchange

### ISSUE THREAD

**Issue by PawelSzymanski89** [przelewy24/p24-mobile-lib-flutter#14]
Why method exchange from class with implementation of GooglePayTrnRegistrar is not exectuing when  P24SDK.googlePay was called?

**Comment by mareklat** [comment]
Does it work on sandbox environment? Error code OR_BIBED_07 means that your application need to be reviewed by Google. To work on production you need stup your app. See Google docs: https://pay.google.com/business/console/

**Comment by PawelSzymanski89** [comment]
Do you mean review of button and enable Google Pay in integrations? It's done.

**Comment by mareklat** [comment]
Yes, I mean integration with Google Pay. If the same type of error appears, I would suggest making sure that the application has been correctly associated with the Google Pay API. For most of our partners who had these types of issues, the error was here.

This kind of error is usually on the application - Google line. If everything is configured correctly GooglePayTrnRegistrar should be called
